packages/cli-server-api/package.json

```
@@ -1,21 +1,20 @@
 {
   "name": "@react-native-community/cli-server-api",
-  "version": "16.0.2",
+  "version": "13.5.1",
```

```suggestion
  "version": "16.0.2",
```

let's keep this one with sync other package